src/dotenv_vault/__init__.py

```diff
@@ -1 +1 @@
-from .main import load_dotenv
+from .main import os, load_dotenv
```

Comment:
@motdotla , we don't need to explicitly `import os` but import as a submodule

Reply:
@motdotla can you PR this? I added in `__all__` so that when people do `from dotenv_vault import *`, it will only import necessary things. See updated readme .